Docker/entry.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I would add an issue for later to add hadolint + shellcheck to CI to lint this script and the Dockerfile, see [this sui-ops PR](https://github.com/MystenLabs/sui-operations/pull/35).

Reply:
Issue opened here https://github.com/MystenLabs/narwhal/issues/277